tests/suites/os/suite.js

```diff
@@ -312,6 +312,35 @@ module.exports = {
 			'Device should be reachable over local SSH connection'
 		)
 
+		await test.resolves( 
+			this.utils.waitUntil(async () => {
+				this.log("Confirming balena engine is running and healthy...");
+				let engine = await this.context
+				.get()
+				.worker.executeCommandInHostOS(
```

Comment:
Should we use `waitForServiceState` here?

Reply:
Also `this.context.get()` is obsolete, you can just use `this.worker.executeCommandInHostOS()`.